certificate/database.go

```diff
@@ -459,6 +473,46 @@ func getCurrentTrust(certID, issuerID int64) (int64, error) {
 	return trustID, nil
 }
 
+func getCurrentTrustIDForCert(certID int64) (int64, error) {
+
+	var trustID int64
+
+	row := db.QueryRow("SELECT id FROM trust WHERE cert_id=$1 AND is_current=TRUE", certID)
+
+	err := row.Scan(&trustID)
+
+	if err != nil {
+
+		if err == sql.ErrNoRows {
+			return -1, nil
+		} else {
+			return -1, err
+		}
+	}
+
+	return trustID, nil
+}
```

Comment:
I will have to review all the code :smile: , or I might as well make an offer to buy you the Duvel brewery :beers: